src/index.coffee

```diff
@@ -9,7 +9,7 @@ module.exports = (env, callback) ->
 
     getPages: (contents) ->
       env.helpers.contents.list(contents).filter (content) ->
-        content instanceof env.plugins.MarkdownPage
+        content instanceof env.plugins.MarkdownPage unless content.metadata?.noindex
```

Comment:
I wrote this check at a time I was unfamiliar with Wintersmith internals and thought sometimes `page.metadata` would be undefined. And after reading the whole `markdown` plugin, I noticed that `page.metadata` is always an object and so my check is unnecessary.

That is why I removed it this morning in 0891c4099f437e0a94c3f69c7de0d1d84242e2e7 (and this change is covered with a markdown page without any metadata).